src/Microsoft.AspNet.SignalR.SqlServer/TaskAsyncHelper.cs

```diff
@@ -0,0 +1,1194 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
Removed unused methods in TaskAsyncHelper.cs, but still have 20+ methods in TaskAsyncHelper.cs are used in SqlServer.